ItalianTranslation_PlanetCoaster.xml

```diff
@@ -3019,188 +3019,188 @@
 		<entry key="BuildingPartName_WS_Sign_B_WantedPoster" translation="Poster da Ricercato"/>
 		<!--C10_Blueprints-->
 		<entry key="Blueprints_VNTRBlueprint01" translation="Argentum"/>
-		<entry key="Blueprints_VNTRBlueprint01Desc" translation="A layout for the F25 coaster&#10;Frontier's 25th Anniversary"/>
+		<entry key="Blueprints_VNTRBlueprint01Desc" translation="Uno schema per il coaster F25&#10;25° Anniversario Frontier"/>
 		<!--C10_Coasters-->
-		<entry key="Coaster_Type_VNTR" translation="Launched - Drive Tyre"/>
+		<entry key="Coaster_Type_VNTR" translation="Lanciato - A pneumatici- "/>
 		<entry key="Coaster_VNTR" translation="F25"/>
-		<entry key="Coaster_VNTR_Desc" translation="Drive Tyre Launched coaster that includes two different drive tyre mechanisms for a variable speed launch.&#10;Frontier's 25th Anniversary"/>
-		<entry key="TrackElementDesc_FrictionWheelLift_VNTR" translation="Propels coasters up a slope or along the track at moderate speeds.&#10;&#10;This track element can be used before the Drive Tyre Launch to create a dual stage launch."/>
-		<entry key="TrackElementDesc_TireLaunch_Booster_VNTR" translation="Propels coasters along the track at a faster speed than the standard Friction Wheel Lift.&#10;&#10;This track element can be used after the Friction Wheel Lift to create a dual stage launch."/>
-		<entry key="TrackElementName_FrictionWheelLift_VNTR" translation="Friction Wheel Lift"/>
-		<entry key="TrackElementName_TireLaunch_Booster_VNTR" translation="Drive Tyre Launch"/>
-		<entry key="TrackElementTool_FrictionWheelLift_VNTR" translation="&lt;font color='accent'&gt;Friction Wheel Lift&lt;/font&gt;&#10;Propels coasters up a slope or along the track at moderate speeds.&#10;&#10;This track element can be used before the Drive Tyre Launch to create a dual stage launch."/>
-		<entry key="TrackElementTool_TireLaunch_Booster_VNTR" translation="&lt;font color='accent'&gt;Drive Tyre Launch&lt;/font&gt;&#10;Propels coasters along the track at a faster speed than the standard Friction Wheel Lift.&#10;&#10;This track element can be used after the Friction Wheel Lift to create a dual stage launch."/>
+		<entry key="Coaster_VNTR_Desc" translation="Coaster lanciato a pneumatici che include due differenti meccanismi a pneumatico per una velocità di lancio variabile.&#10;25° Anniversario Frontier"/>
```

Comment:
"25° Anniversario Frontier" secondo me suona male, perchè non "25° Anniversario **di** Frontier"?

Reply:
fatto

Reply:
@RaydenMK magari hai fatto ma non hai committato o pushato le modifiche nel tuo RaydenMK/PlanetCoasterItalian/beta quindi in questa PullReq non appaiono